Nvli 333 mirador ui

safallia-joseph opened the pull request:
Recreated the "Resource" feature from dev.

Commit:
NVLI-40 - Modify the solrdoc id field name in nvli_custom_mirador_metadata_alter

Commit:
Fix merge conflict

Commit:
NVLI-40 - Recreate resource feature from dev server
Adds the mirador mode view permission to anonymous user
Removes the title from mirador view mode

Commit:
NVLI-333 - Add a template for mirador view mode
In the remplate only print the content value, Remove the title field and author info

piyuesh23 commented on docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml:
```diff
@@ -58,9 +58,10 @@ content:
+      fivestar_widget: modules/fivestar/widgets/basic/basic.css
     third_party_settings: {  }
-    type: fivestar_rating
+    type: fivestar_stars
```
This doesn't look related.

safallia-joseph replied:
yeah! I recreated the "Resource" feature from dev. It had this much changes.
Maybe somebody directly modified in dev, and did not exported as feature.
Or change made in code and not synced in dev!

safallia-joseph replied:
@piyuesh23 removed it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154

piyuesh23 commented on docroot/modules/features/resource/config/optional/views.view.mirador_view.yml:
```diff
@@ -5,6 +5,7 @@ dependencies:
     - core.entity_view_mode.node.mirador
     - node.type.resource
     - user.role.administrator
+    - user.role.anonymous
```
Change the view access to permission based. Users having access to view published content should be able to access mirador view.

safallia-joseph replied:
ok

safallia-joseph replied:
modified it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154

Commit:
NVLI-333 - Recreate resource feature from dev
Add permission 'access content' for accessing view mode mirador
Remove the unwanted changes in the resource feature
